### DISCUSSION TITLE
Still the home screen and then nothing.

### DISCUSSION THREAD

**Discussion by Cdcomtom** [Blackymas/NSPanel_HA_Blueprint#2215] [Q&A]
Hi guys, after flashing the new firmware to 4.3.9, I only have the home screen and even after setting in the HA Blueprint template, only the time and temperature are displayed. has anyone had the same problem.

![20240804_161119](https://github.com/user-attachments/assets/43245cbc-1c6b-4401-a981-9b13809b0b69)

**Comment by edwardtfn** [comment]
On this screen I see the Blueprint isn't connected to the panel.
Have you selected the panel in the Blueprint settings?
Is your Blueprint also v4.3.9?

**Comment by Cdcomtom** [comment]
Yes, a template uploaded today.
![20240804_205228](https://github.com/user-attachments/assets/cbe643b0-b8d0-4294-8701-8b87b7ca97c9)

**Comment by Cdcomtom** [comment]
But in the diagnostics it says unknown to me.
![20240804_205952](https://github.com/user-attachments/assets/4946f28a-19b9-41bd-b3eb-0d4f4c893aa5)
